index.js

```diff
@@ -0,0 +1,31 @@
+const moves = document.getElementById("moves-count");
+const timeValue = document.getElementById("time");
+const startButton = document.getElementById("start");
+const stopButton = document.getElementById("stop");
+const gameContainer = document.querySelector(".game-container");
+const result = document.getElementById("result");
+const controls = document.querySelector("controls-container");
+let cards;
+let interval;
+;
+
+
+const emojis=["🥶","🥶","😈","😈","👿","👿","😡","😡","🥵","🥵","🤬","🤬","🎃","🎃","🤮","🤮","😰","😰","😨","😨"];
+let shuf_emojis = emojis.sort(function() {
+    return (Math.random() > 0.5) ? 1 : -1;
+});
+
+ for( let i = 0; i<emojis.length; i++){
+    let box = document.createElement('div');
+    box.className = 'item';
+    box.innerHTML = shuf_emojis[i];
+    box.onclick = function() {
+        this.classList.add('boxOpen')
```

Comment:
The requirements for this week include `on click, the card flips from showing the backside pattern to the picture and vice versa.` Your project lacks the functionality to flip the card back.

Reply:
i don't know how to do it but i hope figure it out

Reply:
In your `onClick` function, you need to check whether the box is open. If yes, you close it. If not, you open it. Check out the functions available for `classList` here: https://developer.mozilla.org/en-US/docs/Web/API/Element/classList; you might find something useful.

Reply:
hi again i did it can you check please?

Reply:
Very good 👍🏻